README.md

```diff
@@ -9,149 +9,39 @@
-    <a href="docs/images/lbf_results/2s-8x8-2p-2f-coop_rec_mappo.png">
-        <img src="docs/images/lbf_results/2s-8x8-2p-2f-coop_rec_mappo.png" alt="Mava ff mappo tiny 2ag" width="30%" style="display:inline-block; margin-right: 10px;"/>
-    </a>
-    <a href="docs/images/lbf_results/15x15-4p-3f_rec_mappo.png">
-        <img src="docs/images/lbf_results/15x15-4p-3f_rec_mappo.png" alt="Mava ff mappo small 4ag" width="30%" style="display:inline-block; margin-right: 10px;"/>
-    </a>
-    <br>
-    <div style="text-align:center; margin-top: 10px;"> Mava Recurrent MAPPO performance on the <code>2s-8x8-2p-2f-coop</code>, and <code>15x15-4p-3fz</code> Level-Based Foraging tasks.</div>
-</p>
-
-### 🧨 Steps per second experiments using vectorised environments
-
-Furthermore, we illustrate the speed of Mava by showing the steps per second as the number of parallel environments is increased. These steps per second scaling plots were computed using a standard laptop GPU, specifically an RTX-3060 GPU with 6GB memory.
-
-<p align="center">
-    <a href="docs/images/speed_results/mava_sps_results.png">
-        <img src="docs/images/speed_results/mava_sps_results.png" alt="Mava sps" width="55%"/>
-    </a>
-    <a href="docs/images/speed_results/ff_mappo_speed_comparison.png">
-        <img src="docs/images/speed_results/ff_mappo_speed_comparison.png" alt="Mava ff mappo speed comparison" width="39.33%" style="display:inline-block; margin-right: 10px;"/>
-    </a>
-    <br>
-    <div style="text-align:center; margin-top: 10px;"> Mava steps per second scaling with increased vectorised environments and total training run time for 20M environment steps.</div>
-</p>
-
-## Code Philosophy 🧘
-
-The current code in Mava is adapted from [PureJaxRL][purejaxrl] which provides high-quality single-file implementations with research-friendly features. In turn, PureJaxRL is inspired by the code philosophy from [CleanRL][cleanrl]. Along this vein of easy-to-use and understandable RL codebases, Mava is not designed to be a modular library and is not meant to be imported. Our repository focuses on simplicity and clarity in its implementations while utilising the advantages offered by JAX such as `pmap` and `vmap`, making it an excellent resource for researchers and practitioners to build upon.
+- 🖥️ **Multi-agent Sebulba**: We maintain a sebulba versions of some of our algorithms allowing for speed ups even with non-Jax environments.
+- ⚡ **Blazingly fast experiments**: All of the above allow for very quick runtime for our experiments, especially when compared to other non-JAX based MARL libraries.
 
 ## Installation 🎬
 
-At the moment Mava is not meant to be installed as a library, but rather to be used as a research tool.
-
-You can use Mava by cloning the repo and pip installing as follows:
+At the moment Mava is not meant to be installed as a library, but rather to be used as a research tool. We recommend cloning the Mava repo and pip installing as follows:
```

Comment:
We can squash a lot of the commit history, I can look into doing this

Reply:
So I've squashed everything before jax mava in this branch: chore/squash

What I'd still like to do is squash all commits between our merge commits (as we still have almost 2k commits in that branch). This is quite tricky so might only be able to do this next year, but it would give us a really nice linear commit history (like jumanji) and reduce our number of commits substantially without losing any info about the commits themselves which is nice.

Reply:
This sounds good, I think for now we can merge the readme update and then early next year we can squash the history for easy cloning in the future.